src/components/shared/VirtualList.js

```diff
@@ -170,6 +177,8 @@ type VirtualListProps = {|
   +disableOverscan: boolean,
   +columnCount: number,
   +containerWidth: CssPixels,
+  // Sometimes we want to force a rerender, for example when the width changes.
+  +forceRender?: number,
```

Comment:
I hesitated between this, or just forwarding `containerWidth` down to the row. I decided to use this approach instead, in case we want a `generation`-like approach in another component in the future.

Reply:
I also thought of using %ages to define the network markers, that would have make resizes fluid. But there's some specific behavior I'd like to keep and that needs pixels (especially that I want at least 1px for 1 existing phase).

Reply:
And actually I used it even more in my newer version.